skimage/filters/thresholding.py

```diff
@@ -135,10 +151,14 @@ def wrapper(im):
                            'Minimum': thresh(threshold_minimum),
                            'Otsu': thresh(threshold_otsu),
                            'Triangle': thresh(threshold_triangle),
-                           'Yen': thresh(threshold_yen)})
+                           'Yen': thresh(threshold_yen),
+                           'Local': thresh(threshold_local),
+                           'Niblack': thresh(threshold_niblack),
+                           'Sauvola': thresh(threshold_sauvola),
+                           'Phansalkar': thresh(threshold_phansalkar)})
```

Comment:
Even if Phansalkar will not be accepted, I think it would be helpfull, if the most available filters are included in this function.